source/clear-linux/guides/maintenance/validate-signatures.rst

```diff
@@ -1,17 +1,19 @@
 .. _validate-signatures:
 
-Validate signatures
-###################
+Validating signatures
```

Comment:
@intelkevinputnam. Minor revision request. Can you revert to original header here? 
We're trying to keep active language in headers too.  Otherwise, LGTM. Thanks for your work on this.

Reply:
Sure. Though this convention makes it difficult to refer to these sections elsewhere without having to change the wording.